src/components/CreateNote/CreateNoteContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+
+import { useFieldValue } from 'hooks';
+
+import { CreateNote } from './CreateNote';
+
+interface ICreateNote {
+  createNote: (title: string, description: string) => void;
+}
+
+export const CreateNoteContainer = ({ createNote }: ICreateNote) => {
+  const [isOpen, setOpen] = useState(false);
```

Comment:
also please add more descriptive naming
```suggestion
  const [isOpen, setOpen] = useState<boolean>(false);
```